### PR TITLE
fix: proxy config unmarshal mesh config failed when meshId is number

### PR DESCRIPTION
**Pull request by chinaran**
**Please provide a description of this PR:**
When install Istio with number meshId, like 222222, istio-proxy will parse error and can't run.

    2022-04-22T03:41:51.570480Z error failed to get proxy config: failed to unmarshal mesh config from file [defaultConfig:
    discoveryAddress: istiod-1-12.istio-system.svc:15012
    extraStatTags:
    - mesh_id
    meshId: 222222
    proxyMetadata:
    ISTIO_META_DNS_AUTO_ALLOCATE: "true"
    ISTIO_META_DNS_CAPTURE: "true"
    ...

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration Infrastructure
- [ ] Docs
- [X] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by zirain** [comment]
@chinaran you should apply to join the org.

**Comment by chinaran** [comment]
@zirain so, is this pr waiting for istio/wg-environments-maintainers code review now?

**Comment by zirain** [comment]
yes

**Comment by chinaran** [comment]
> yes

Thanks.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**
